ark/analysis/spatial_analysis.py

```diff
@@ -44,11 +45,7 @@ def calculate_channel_spatial_enrichment(dist_matrices_dict, marker_thresholds,
     """
 
     # Setup input and parameters
-    if included_fovs is None:
-        included_fovs = list(set(all_data[fov_col]))
-        num_fovs = len(included_fovs)
-    else:
-        num_fovs = len(included_fovs)
+    included_fovs, num_fovs = test_utils.get_fov_info(all_data, fov_col, included_fovs)
```

Comment:
I think this is over-engineering. Especially given that we'd only use this inside this script, I think it adds clutter to have a function that does something this simple